src/js/tabs/motors.js

```diff
@@ -579,7 +579,7 @@ TABS.motors.initialize = function (callback) {
         // Amperage
         function power_data_pull() {
             motorVoltage.text(i18n.getMessage('motorsVoltageValue', [FC.ANALOG.voltage]));
-            motorMahDrawingElement.text(i18n.getMessage('motorsADrawingValue', [FC.ANALOG.amperage.toFixed(2)]));
+            motorMahDrawingElement.text(i18n.getMessage('motorsADrawingValue', [FC.ANALOG.amperage?.toFixed(2)]));
```

Comment:
What does this change?

Reply:
Need some more time here. Put the PR in draft. The change is not needed anymore, but was suppressing an error where the variable was not yet set,